Runtime/Client/LootLockerServerRequest.cs

```diff
@@ -150,8 +150,141 @@ public static T InputUnserializableError<T>() where T : LootLockerResponse, new(
         {
             return Error<T>("Method parameter could not be serialized");
         }
+
+        /// <summary>
+        /// Construct an error response because an rate limit has been hit
+        /// </summary>
+        public static T RateLimitExceeded<T>(string method, int secondsLeftOfRateLimit) where T : LootLockerResponse, new()
+        {
+            return Error<T>(string.Format("You are sending too many requests and are being rate limited for %d seconds. Call was to endpoint %s", secondsLeftOfRateLimit, method));
+        }
+    }
+
+
+
+    #region Rate Limiting Support
+
+    public class RateLimiter
+    {
+        /* -- Configurable constants -- */
+        // Tripwire settings, allow for a max total of n requests per x seconds
+        protected const int TripWireTimeFrameSeconds = 60;
+        protected const int MaxRequestsPerTripWireTimeFrame = 280; // The backend rate limit is 300 requests per minute so we'll limit it slightly before that
```

Comment:
Remove the comment telling what the back-end rate limit is.

Reply:
Fixed